### ISSUE TITLE
Unable to create fixed discounts for each invoice/bill item.

### ISSUE THREAD

**Issue by rihards-simanovics** [akaunting/akaunting#2102]
I am currently unable to create a negative price for the line items (let's say as a discount, please see below), only a negative quantity (which I believe is counterintuitive).
I would suggest making the price validation accept negative so long as the overall bills/invoice cost (total) stays positive.

### Steps to reproduce the issue
Enter an item price on the bill or invoice that is in the negative see the example below:

### Expected result
A negative value for the price should be allowed through the validation.
![image](https://user-images.githubusercontent.com/52106095/121701093-73ccbe80-cac8-11eb-8307-ac2bf87514f1.png)

### Actual result
a negative value for price fails serverside validation
![image](https://user-images.githubusercontent.com/52106095/121701183-8e069c80-cac8-11eb-8225-c14043ef643a.png)

### System information (Akaunting, PHP versions)
Version 2.1.16
PHP 7.4.20 FPM served by Nginx
FastCGI Support enabled

### Additional comments
I think you could make sure that the overall or the total price on the bill or invoice value is positive, although I would suggest allowing even that. Some companies may need to make refunds to their customers and because it is still considered an invoice the negative value should carry. Now I understand that it may confuse many validation errors, so what I would say for now is to have a system that allows a negative item value but not the bitt/invoice total.

**Comment by cuneytsenturk** [comment]
Akaunting already has line discount, You can read this [documentation](https://akaunting.com/docs/user-manual/settings/localisation), You can change discount location field.

**Comment by enieber** [comment]
@rihards-simanovics  thanks for you report, this is not a bug, it is a feature

**Comment by rihards-simanovics** [comment]
Thanks @enieber, as I've created this issue a long time ago and @cuneytsenturk (genuine thanks, at the time I found the problem) has responded very timely with a helpful solution at the time, I'm finding this not to be an issue any longer.

That being said, by now, I have moved to a an accounting solution with more features, better security, and a more user friendly interface.

Additionally I thank you for your timely and and long awaited comment, which has triggered GitHub to send me more spam to my mailbox, it's very appreciated.
